spring-integration-jms/src/main/java/org/springframework/integration/jms/ChannelPublishingJmsMessageListener.java

```diff
@@ -518,6 +519,10 @@ public ErrorMessage buildErrorMessage(Throwable throwable) {
 			return super.buildErrorMessage(null, throwable);
 		}
 
+		MessagingTemplate getMessagingTemplate() {
```

Comment:
Just curious: what made you to add this method?

And if it is here already any way how about to make it `protected` ?